### ISSUE TITLE
Port xenial onboard patches to bionic

### ISSUE THREAD

**Issue by codygarver** [elementary/os-patches#40]
In the past I have tweaked onboard to not look so bad out of the box but I have not had the chance to do this yet because I'm blocked by elementary/gala#293

**Comment by codygarver** [comment]
Blocked by elementary/gala#293

**Comment by davidmhewitt** [comment]
Fix merged into gala, marking as unblocked.

**Comment by danrabbit** [comment]
Unblocking since the Gala fix was merged

**Comment by codygarver** [comment]
Instead of messing with patches I have set the default theme to `Nightshade` elementary/default-settings@15d9a04
